tools/cachemgr.cc

```diff
@@ -693,6 +693,7 @@ read_reply(int s, cachemgr_request * req)
             }
 
             istate = isActions;
+            __attribute__((fallthrough));
```

Comment:
gcc and clang support it, and given that icc strives to be gcc-compatible, I expect it supports it as well. This should cover any compilers we care for. We still have some shims for sunstudio but I a not aware of anybody using it for squid.

A comment will not work, because compilers will ignore it.
I can codemod to a comment if I disable that specific check with a -Wno-* option.
I'd like to leave __attribute__ in and switch to [[attribute]] when we can rely on c++17

Reply:
Assuming __attribute__ is not standard in C++, please avoid that construct even if your compiler and all tested compilers support it.

Reply:
ok